.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
```

Comment:
I'd recommend adding something like this to the starting of this file -
```yaml
ci:
  autoupdate_commit_msg: "chore: update pre-commit hooks"
  autofix_commit_msg: "style: pre-commit fixes"
```

This will load `pre-commit.ci` with better commit messages (whenever we turn it on for this repository). Let us know if you want us to turn it on for this repository.

Reply:
I was going to ask you about it on Slack haha, please feel free to turn it on 😄

Reply:
Done! The badge should get green soon - 

[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/pybamm-team/pybamm.org/main.svg)](https://results.pre-commit.ci/latest/github/pybamm-team/pybamm.org/main)